language/move-analyzer/editors/code/src/commands/lsp_command.ts

```diff
@@ -1,24 +1,42 @@
 import type {
```

Comment:
Could you please conform to the existing indentation scheme to avoid unnecessary changes?

Reply:
done